### ISSUE TITLE
Provide a link for contribution instructions in  the "about tab"

### ISSUE THREAD

**Issue by chmodsayshello** [minetest/minetest#15281]
### Problem

In the about tab, there's a list of contributors, yet no information on how to get involved with minetest.

### Solutions

Add a button under said list which opens https://www.minetest.net/get-involved/ in a web-browser upon being clicked.

### Alternatives

-

**Comment by appgurueu** [comment]
I don't think this is very important: We already have a link to minetest.net; from there on the "get involved" page is only one click away.

Besides, someone with intent to contribute and basic search engine skills should be able to find e.g. this GitHub repo relatively quickly.

**Comment by cx384** [comment]
> only one click away

This may not seem much, but one extra click is a big difference, especially when it comes to websites.

The get-involved page also links to game/mod development things, and to communication platforms like the Forum, IRC, Discord, Matrix. Imo, it wouldn't hurt to attract more people this way.
A main menu redesign is planed/going on, so the about tab should probably also be rethought eventually.
